lib/components/Input/input-phone-number/index.tsx

```diff
@@ -40,11 +40,6 @@ const InputPhoneNumberContent = forwardRef<
             fillAddonBorderColor,
             countryCodes,
             shortCode,
-            placeholder = "00 0000 0000",
-            formatProps = {
-                format: "## #### ####",
-                mask: "",
-            },
```

Comment:
Just to make sure, do we have consumer need this formatProps? Can we make this optional?

Reply:
This props is already optional from default, i checked in my project i can pass in formatProps from my side